CollapseLauncher/XAMLs/MainApp/Pages/HomePage.GameManagement.cs

```diff
@@ -556,8 +555,47 @@ private void CancelInstallationProcedure(object sender, RoutedEventArgs e)
                 break;
         }
     }
+
+    private void PostInstallProcedure()
+    {
+        if (!CurrentGameProperty.GameVersion.IsGameInstalled()) return;
+
+        var behaviour = CurrentGameProperty.GameInstall.PostInstallBehaviour;
+        switch (behaviour)
+        {
+            case PostInstallBehaviour.Nothing:
+                break;
+            case PostInstallBehaviour.StartGame:
+                StartGame(null, null);
+                break;
+            case PostInstallBehaviour.Hibernate:
+                Process.Start(new ProcessStartInfo("C:\\Windows\\System32\\shutdown.exe", "/h")
+                {
+                    CreateNoWindow = true,
+                    UseShellExecute = false,
+                });
+                break;
+            default:
```

Comment:
I think its better to put this as its own case so that it doesn't behave unexpectedly when the code modified in the future. This also speeds up a bit when user doesn't select anything (Nothing)
something like 
```csharp
case PostInstallBehavior.Restart:
case PostInstallBehavior.Shutdown:
   // Implementation
   break;   
// you can eliminate default: here
```